drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
-	callback    driverapi.DriverCallback
```

Comment:
okay. Fair enough. I will look forward for the subsequent PRs.
The only intention of my question is to make sure we don't end up in a design to include the remote drivers statically in libnetwork project.